README.md

```diff
@@ -35,27 +35,23 @@ And review the following guides:
 
 To include the SDK binaries in your own project, follow these steps.
 
-For the purpose of setup, your project's root folder (where you can find your `.xcodeproj` file) will be referred to as `root`
+For the purpose of setup, your project's root folder (where you can find your `.xcodeproj` file) will be referred to as `root`.
 
 ### 1. Download binaries
 
-Download the following zips:
```

Comment:
❤
update to the runbook?

Reply:
Will do afterwards. I will update Android too